crates/sui-types/src/deny_list_v2.rs

```diff
@@ -20,6 +20,31 @@ use serde::{Deserialize, Serialize};
 use std::collections::{BTreeMap, BTreeSet};
 use std::fmt;
 
+// esitmate of the size of the objects read for a deny list check
```

Comment:
This is something that we should fix forward but based on my understanding, the size of objects (in our scale) does not really matter that much when it comes to db read cost. Once there is a read (regardless of the result) there is some non-trivial fixed cost.

Reply:
How do you want to cost this then? 
I really just need some number